Sources/XcodeGraphMapper/Mappers/Targets/PBXTargetMapper.swift

```diff
@@ -334,28 +337,6 @@ struct PBXTargetMapper: PBXTargetMapping {
         return sources.filter { $0.path.fileExtension == .playground }.map(\.path)
     }
 
-    /// Reads and parses a plist file into a `[String: Plist.Value]` dictionary.
-    private func readPlistAsDictionary(
```

Comment:
for some reason, when mapping a generated `Tuist.xcworkspace` in `tuist/tuist`, this would always throw with the same error as we've seen in the past: https://github.com/tuist/tuist/issues/7018

Since we don't strictly need the plist dictionary need, I decided to, at least for now, return a plain path reference instead of parsing the plist.